lib/instrumentation/span.js

```diff
@@ -219,6 +219,12 @@ Span.prototype._encode = function (cb) {
       outcome: self.outcome
     }
 
+    // if a valid sample rate is set (truthy or zero), set the property
+    const sampleRate = self.sampleRate
+    if (sampleRate || sampleRate === 0) {
```

Comment:
nit: Perhaps use `if (sampleRate !== null)` given that the sampleRate getting returns `null` for an invalid value.